via/services/youtube.py

```diff
@@ -39,6 +41,15 @@ def get_video_id(self, url):
 
         return None
 
+    def get_transcript(self, video_id):
```

Comment:
It seems like we'd _really_ benefit from some caching here, or are we hoping this will end up getting cached by Cloudflare? I couldn't see any specific handling or headers from the view to help with that.

Thinking about it, with our security model I'm not sure that it's possible that you could?

Reply:
The plan in future is to add caching (in fact long-term storage) by persisting the transcripts in a Postgres DB that will be added to Via.

Even with this Postgres-based caching I can imagine Cloudflare-based caching also being useful in addition to keep traffic off our servers, but I don't think it's possible because requests to this API are currently authenticated using an `Authorization` header. I think we'd have to remove the authentication (and make this an open API) in order to benefit from Cloudflare caching.

> Thinking about it, with our security model I'm not sure that it's possible that you could?

Exactly. You could make an argument that this API doesn't need authentication and could better benefit from caching instead. But when this was discussed authentication won the day